Devscord.DiscordFramework/Framework/Commands/Responses/ResponsesManager.cs

```diff
@@ -49,5 +49,10 @@ public static string CurrentVersion(this ResponsesService responsesService, Cont
         {
             return responsesService.ProcessResponse("CurrentVersion", contexts, new KeyValuePair<string, string>("version", version));
         }
+
+        public static string PrintHelp(this ResponsesService responsesService, string help)
+        {
+            return responsesService.ProcessResponse("PrintHelp", new KeyValuePair<string, string>("help", help));
```

Comment:
Lepiej przez metode -> tak jak w przypadku innych odpowiedzi

opcja przez nazwe odpowiedzi (tak jak tutaj) jest możliwa, ale jednak wolałbym to robić przez opcje z metodą, wtedy kod będzie odporniejszy na drobne błędy
 
i w przypadku zmiany nazwy odpowiedzi nie będzie szansy na skutki uboczne

Reply:
Nie rozumiem za bardzo, jest tak samo jak inne. Czym się różni?

Reply:
Aaa źle spojrzałem, jest dobrze